meta-iot2000-bsp/files/iot2000/sw-description.bbin

```diff
@@ -8,21 +8,23 @@ software =
                 {
                         images: (
                         {
-                                filename = "${image}";
+                                filename = "${image}-enc";
                                 sha256 = "@${image}";
                                 device = "/dev/mmcblk0p2";
-				compressed = true;
+                                compressed = true;
+                                encrypted = "@${image}-status";
                         }
                         );
                 };
                 platform1:
                 {
                         images: (
                         {
-                                filename = "${image}";
+                                filename = "${image}-enc";
                                 sha256 = "@${image}";
                                 device = "/dev/mmcblk0p3";
-				compressed = true;
+                                compressed = true;
```

Comment:
code in swupdate suggests that they do not support images that are both encrypted and compressed. did it work for you?

Reply:
He still need to test this on baord.

Reply:
Does it mean that we no longer support non-encrypted SWUs?
Should we instead use a different bbin file when encryption is enabled?

Reply:
Excerpt from proposed final bbin looks like:
```
+                                compressed = false;
+                                encrypted = @${image}-status;
```
So, if we have SWUPDATE_ENCRYPTION set to 1 in local.conf encrypted swu will be supported, otherwise un-encrypted swu will be supported.